File: app/src/main/java/com/simplemobiletools/calculator/operation/MultiplyOperation.java
package com.simplemobiletools.calculator.operation;

import com.simplemobiletools.calculator.operation.base.BinaryOperation;
import com.simplemobiletools.calculator.operation.base.Operation;

public class MultiplyOperation extends BinaryOperation implements Operation {
    protected MultiplyOperation(double baseValue, double secondValue) {
        super(baseValue, secondValue);
    }

    @Override
    public double getResult() {
        return baseValue * secondValue;
    }
}

File: app/src/main/java/com/simplemobiletools/calculator/operation/MultiplyOperation.kt
package com.simplemobiletools.calculator.operation

import com.simplemobiletools.calculator.operation.base.BinaryOperation
import com.simplemobiletools.calculator.operation.base.Operation

class MultiplyOperation(baseValue: Double, secondValue: Double) : BinaryOperation(baseValue, secondValue), Operation {

    override fun getResult() = baseValue * secondValue
}
